src/rules/no-trailing-spaces.js

```diff
@@ -0,0 +1,22 @@
+var fs = require('fs');
+var rule = 'no-trailing-spaces';
+
+function noTrailingSpaces(parsedFile, fileName) {
```

Comment:
The gherkin parser actually removes trailing spaces, which is why we have to read the file in a different way. The good thing is that this is also much simpler than using the tree that the parser generates, but we sacrifice consistency.